spec/models/exception_hunter/error_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+module ExceptionHunter
+  describe Error do
+    subject { build(:error) }
+
+    describe 'validations' do
+      it { is_expected.to validate_presence_of(:class_name) }
+    end
+
+    describe 'before validation callback' do
```

Comment:
How about
```suggestion
    describe 'occurred at' do
```

A small change but it discourages people from putting generic tests on one place